tests/foreman/cli/test_capsule.py

```diff
@@ -382,3 +386,25 @@ def test_positive_enable_bmc():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.skip_if_not_set('fake_capsules')
+@pytest.mark.tier1
+def test_positive_search_nondefault_proxy(request, target_sat):
+    """Search non-default proxy  with the home proxy
+
+    :id: caf51662-6b4e-11ed-baba-2b9d7b368002
+
+    :expectedresults: Non-default proxy can be searched
+
+    :BZ: 2077824
+
+    :customerscenario: true
+    """
+    proxy = _make_proxy(request, target_sat, options={'name': gen_string('alpha')})
+    path = target_sat.api.Capsule().path() + '?search=id+%21%3D+1'
+    response = client.get(path, auth=get_credentials(), verify=False)
```

Comment:
Why does this test is written in CLI test module rather than API?

Asked because all the steps and verification you have been dong here with API.

Reply:
The API counterpart would be in `tests/foreman/api/test_smartproxy.py`

Reply:
yes, there is no separate module present yet for API yet

Reply:
updated